contracts/RMRK/extension/tokenProperties/RMRKTokenProperties.sol

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.16;
+
+import "./IRMRKTokenProperties.sol";
+
+/**
+ * @title RMRKTokenProperties
+ * @author RMRK team
+ * @notice Smart contract of the RMRK Token properties module.
+ */
+abstract contract RMRKTokenProperties is IRMRKTokenProperties {
```

Comment:
I have really grown to like how abstract contracts are useful. Great improvement over the previous version.

Reply:
For our use case of creating a standard + minimal + full implementation, it's a game changer.
Thanks!